packages/logger/index.ts

```diff
@@ -1,27 +1,42 @@
-import pino from "pino";
+import debug from "debug";
 
-const logger = pino({
-  level: process.env.NEXT_PUBLIC_PINO_LOG_LEVEL || "info",
-  browser: {
-    write(obj) {
-      const msg = "msg" in obj && obj.msg;
-      console.warn(
-        `Invalid use of @oakai/logger, use logger/browser, logMessage=${msg}`,
-      );
-    },
-  },
-  formatters: {
-    // Pino logs the level as a syslog number, so make sure
-    // it sends error/warn/info etc instead
-    level: (label) => {
-      return { level: label.toUpperCase() };
-    },
-  },
-  // Pino's default key of `time` is ignored by datadog, so explicitly set
-  // `timestamp`
-  timestamp: () => `,"timestamp":"${new Date(Date.now()).toISOString()}"`,
-});
+const baseLogger = debug("ai");
 
-export type Logger = typeof logger;
+type ChildKey =
+  | "aila"
+  | "aila:analytics"
+  | "aila:errors"
+  | "aila:llm"
+  | "aila:moderation"
+  | "aila:persistence"
+  | "aila:prompt"
+  | "aila:protocol"
+  | "aila:rag"
+  | "aila:testing"
+  | "app"
+  | "analytics"
+  | "auth"
+  | "admin"
+  | "chat"
+  | "consent"
+  | "cloudinary"
+  | "demo"
+  | "exports"
+  | "ingest"
+  | "feedback"
+  | "fixtures"
+  | "generation"
+  | "inngest"
+  | "judgements"
+  | "qd"
+  | "rag"
+  | "rate-limiting"
+  | "ui"
+  | "testing";
```

Comment:
This list is really the crux of making this logging useful. I haven't given myself a lot of time to split this perfectly, so I'm open to suggestions about how we shake out the different parts of the project

For example, we're treating aila as a package with its own namespace, but we also have categories like `analytics` and `rag` on the main namespace

I'm very happy for these to evolve over time